src/python/setup.py

```diff
@@ -37,12 +37,9 @@
 	maintainer_email = "paul@paul-moore.com",
 	license = "LGPLv2.1",
 	platforms = "Linux",
-	cmdclass = {'build_ext': build_ext},
-	ext_modules = [
+	ext_modules = cythonize([
 		Extension("seccomp", ["seccomp.pyx"],
 			# unable to handle libtool libraries directly
-			extra_objects=["../.libs/libseccomp.a"],
-			# fix build warnings, see PEP 3123
```

Comment:
I didn't observe any build warnings on Python 3.10.2.  Do we still need this?

Reply:
I ... don't know?  The python bindings went in so long ago I don't recall why it was necessary beyond the comment.

Is it so that the bindings are built with the library statically linked as opposed to using a shared library installed on the system?

Reply:
You shouldn't need `-fno-strict-aliasing` with Python 3. It was an issue with Python 2's C API, where the first fields in an extension type's struct duplicated the fields from PyObject (i.e. the reference count and type pointer). In Python 3, the first field of an extension type's struct is a PyObject.

From the look of it, you've dropped Python 2 support, so you could safely drop `-fno-strict-aliasing` too.

Reply:
Thanks @jhenstridge, mystery solved :)